var/spack/repos/builtin/packages/hpctoolkit/package.py

```diff
@@ -45,17 +46,19 @@ class Hpctoolkit(AutotoolsPackage):
 
     # Options for MPI and hpcprof-mpi.  We always support profiling
     # MPI applications.  These options add hpcprof-mpi, the MPI
-    # version of hpcprof.  Cray needs a separate option because an
-    # external MPI module in packages.yaml doesn't work.
+    # version of hpcprof.  Cray is a separate option for old systems
+    # where an external MPI module doesn't work.
     variant(
-        "cray", default=False, description="Build hpcprof-mpi for Cray systems (requires --dirty)."
+        "cray",
+        default=False,
+        description="Build hpcprof-mpi for Cray systems (may require --dirty).",
```

Comment:
Just a note: using `sticky=True` prevents clingo from accidentaly changing this variant (and require users to change it explicitly if need be). If that's useful we can add that in another PR.

Reply:
I don't understand.

Reply:
https://spack.readthedocs.io/en/latest/packaging_guide.html#sticky-variants

Reply:
Hmm, I'll have to think about this for a while.
I don't feel that I understand all the implications of using `sticky`.
But it's good to know, a useful tool in the toolbox.  Thanks!